crates/aiken/src/cmd/packages/add.rs

```diff
@@ -22,7 +22,7 @@ pub struct Args {
     #[clap(long)]
     pub version: String,
 
-    #[clap(hide = true)]
+    #[clap(hide = true, long)]
```

Comment:
This is actually a bug fix, because the argument is optional but positional at the same time. When we generate the completions, we trigger all possible options and it catched this error.